spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -25,8 +25,7 @@ definitions:
      - code:
             type: u8
             desc: >
-                Signal encoded following RTCM specifications
-                (DF380, DF381, DF382 and DF467).
+                Signal encoded following Swift RTCM specifications. Table Signal and Tracking Mode Identiﬁer.
```

Comment:
Do you have better alternatives to indicate the use of the new table? Something to make this more clear?